### ISSUE TITLE
auto logout

### ISSUE THREAD

**Issue by hajdumozes** [hajdumozes/external-spotify#15]
### Desc

Currently the logout is not available, and the user simply lose access to the spotify api after 1 hour of inactivity.
This should be changed with a manual logout and an auto logout accessibility.

### Subtasks

- [ ] logout functions
- [ ] logout invocations
